libcst/_nodes/expression.py

```diff
@@ -516,9 +516,27 @@ class BaseString(BaseExpression, ABC):
     pass
 
 
+StringQuoteLiteral = Literal['"', "'", '"""', "'''"]
+
+
 class _BasePrefixedString(BaseString, ABC):
-    @abstractmethod
-    def _get_prefix(self) -> str:
+    @property
+    def prefix(self) -> str:
```

Comment:
Can we also use Literal to type prefix? It's also a limited set of strings.

Reply:
Literal makes less sense here, as prefix validity is version-dependent.